tlatools/org.lamport.tlatools/src/pcal/Tokenize.java

```diff
@@ -927,15 +926,15 @@ public static TLAExpr InnerTokenize(PcalCharReader charReader,
               * optimizations that are local to a particular case clause.  *
               *************************************************************/
               { case START :
-                  if (Misc.IsSpace(nextChar))
+                  if (' ' == nextChar || '\f' == nextChar || '\r' == nextChar)

```

Comment:
Same comment as below: Misc is also used by tla2tex, and, thus, should only be replaced iff it is replaced everywhere.